nodejs/lib/util/bufferutils.js

```diff
@@ -7,6 +7,8 @@ this.BufferUtils = (function() {
 
 	BufferUtils.isBuffer = Buffer.isBuffer;
 
+	BufferUtils.toArrayBuffer = function(buf) { return buf; };
```

Comment:
Not sure I like the name of this, but it's not better than `mightBeWordArrayOrBufferToBuffer`.